zipline/pipeline/common.py

```diff
@@ -8,6 +8,8 @@
 CASH_AMOUNT_FIELD_NAME = 'cash_amount'
 COUNT_FIELD_NAME = 'count'
 BUYBACK_ANNOUNCEMENT_FIELD_NAME = 'buyback_date'
+BUYBACK_TYPE_FIELD_NAME = 'buyback_type'
+CASH_AMOUNT_FIELD_NAME = 'cash_amount'
```

Comment:
Whoops - I put it there twice by accident (removed). It's used for dividends, though.